diff -urN shntool-1.2.3/src/Makefile.am shntool-1.2.3.pius/src/Makefile.am
--- shntool-1.2.3/src/Makefile.am	Mon Feb 17 01:25:50 2003
+++ shntool-1.2.3.pius/src/Makefile.am	Mon Apr 28 20:44:51 2003
@@ -19,10 +19,10 @@
 
 install-exec-hook:
 	for mode_alias in $(MODE_ALIASES_ALL) ; do \
-	  test -e $(bindir)/shn$$mode_alias && $(RM) $(bindir)/shn$$mode_alias ; \
+	  test -e $(DESTDIR)$(bindir)/shn$$mode_alias && $(RM) $(DESTDIR)$(bindir)/shn$$mode_alias ; \
 	done ; \
 	for mode_alias in $(MODE_ALIASES) ; do \
-	  test -e $(bindir)/shn$$mode_alias || $(LN_S) $(PACKAGE) $(bindir)/shn$$mode_alias ; \
+	  test -e $(DESTDIR)$(bindir)/shn$$mode_alias || $(LN_S) $(PACKAGE) $(DESTDIR)$(bindir)/shn$$mode_alias ; \
 	done
 
 distclean-local:
